CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+## 3.4.0-SNAPSHOT (2018-10-22)
+
+**Important**: This is an alpha release that provides early access to the following new features:
+- Enhanced support for Spring applications with the new `bugsnag-spring` notifier
+- The ability to report throwables to Bugsnag with the [logback](https://logback.qos.ch/) appender `BugsnagAppender`  
+
+Two artifacts are now available for this platform:
+
+- `bugsnag-java` - intended for plain Java applications
+- `bugsnag-spring` - provides enhanced support for Spring applications
+
+It is recommended that you migrate to `bugsnag-spring` if you develop a Spring application, as it enhances the quantity and quality of error reports which are sent automatically. Full upgrade instructions can be found [here](UPGRADING.md).
+
+No upgrade steps are required for `bugsnag-java` in this release.
+
+* Added `BugsnagAppender` that can report throwables from existing log statements to Bugsnag, using [logback](https://logback.qos.ch/)
+* [Spring] Automatically report exceptions thrown when processing MVC/REST requests
+* [Spring] Automatically report exceptions thrown in scheduled tasks
+* [Spring] Added `BugsnagAsyncConfig` class to simplify capture of uncaught exceptions in async tasks
+* [Spring] Automatically attach request metadata to reports
+* [Spring] Automatically attach Spring version information to reports
+* [Spring] Automatically track sessions for each MVC request
+
+**Deprecation notice**: `Bugsnag.init(...)` is now the preferred way to get an instance of the Bugsnag client
```

Comment:
Let's remove this deprecation and `Bugsnag.init()`, unless there was a technical reason to add it aside from the previous client sharing logic.

Reply:
removed